### PR TITLE
docs: new translate web-components.md

### PR DESCRIPTION
**Pull request by Cnotech**
接https://github.com/vuejs/docs-next-zh-cn/pull/730 ，合并了wxsms的建议

### CODE REVIEW

**Comment by Jinjiang** [comment]
@wxsms any comments here?

File: src/guide/web-components.md [hunk 1]
```diff
@@ -84,48 +83,47 @@ Vue supports creating custom elements using exactly the same Vue component APIs
...
 
 const MyVueElement = defineCustomElement({
-  // normal Vue component options here
+  // 在此提供正常的 Vue 组件选项
   props: {},
   emits: {},
   template: `...`,
 
-  // defineCustomElement only: CSS to be injected into shadow root
+  // defineCustomElement 独有特性: CSS 会被注入到隐式根(shadow root)中
```
Comment (wxsms) [inline src/guide/web-components.md:91]:
```suggestion
  // defineCustomElement 独有特性: CSS 会被注入到隐式根 (shadow root) 中
```

**Review by wxsms — CHANGES_REQUESTED** [review]

**Comment by wxsms** [comment]
> @wxsms any comments here?

most of my comments have been applied

File: src/guide/web-components.md [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
+我们认为 Vue 和 Web Components 大体上是互补的技术。Vue 能很好地解析和创建自定义元素。不论是在将自定义元素整合到已有的 Vue 应用中，还是使用 Vue 构建和分发自定义元素，你都能获得很好的支持。
 
-## 使用 Vue 构建自定义元素
+## 在 Vue 中使用自定义元素
 
-Vue [scores a perfect 100% in the Custom Elements Everywhere tests](https://custom-elements-everywhere.com/libraries/vue/results/results.html). Consuming custom elements inside a Vue application largely works the same as using native HTML elements, with a few things to keep in mind:
+Vue [在 Custom Elements Everywhere 测试中获得了 100% 的完美分数](https://custom-elements-everywhere.com/libraries/vue/results/results.html)。Vue 应用程序中解析出的自定义元素大体上和原生 HTML 元素相同，但需要牢记以下几点：
 
-### Skipping Component Resolution
+### 跳过解析为组件
```
Comment (Jinjiang) [inline src/guide/web-components.md:11]:
```suggestion
### 跳过组件的解析
```

**Pull request merged**